extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -1704,3 +1607,6 @@ An array of objects mapping per-instance feature IDs to property arrays in a fea
     * Added optional `statistics` object which provides aggregate information about select properties within the model
   * Other changes
     * Added `EXT_feature_metadata` extension to the [`EXT_mesh_gpu_instancing`](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/EXT_mesh_gpu_instancing) extension for assigning metadata to instances
+* **Version 2.0.0** September 2021
+  * Renamed `constant` to `offset`, and `divisor` to `repeat`
+  * Removed `statistics` specification, to be provided at the tile level for now, and perhaps a future extension
```

Comment:
I would probably leave out the point about "tile level", since that's really `3DTILES_metadata`-specific.

In the context of this glTF extension by itself, we're just simplifying this extension by reducing scope.

Reply:
Done ✅